pipeline/src/pipeline-validator/jsonschemavalidator.ts

```diff
@@ -0,0 +1,33 @@
+import Ajv from 'ajv'
```

Comment:
pls rename file to `JsonSchemaValidator.ts`

Reply:
x